code/game/objects/items/storage/marine_boxes.dm

```diff
@@ -171,11 +171,6 @@
 	new /obj/item/ammo_magazine/rocket/recoilless/light(src)
 	new /obj/item/storage/backpack/rpg(src)
 
-
-
-
-
-
```

Comment:
this is nice but completely out of the way.

Reply:
originally TX55 has in a box like RR, but Tivi said scrap the box. You uh, want the spaces back?